torch/csrc/dynamo/eval_frame.c

```diff
@@ -745,17 +798,17 @@ static PyObject* _custom_eval_frame(
     return NULL;
   } else if (result != Py_None) {
     DEBUG_TRACE("create cache %s", get_frame_name(frame));
-    extra = create_cache_entry(extra, result);
+    CacheEntry* new_cache_entry = create_cache_entry(cache_entry, result);
```

Comment:
But... you don't end up using cache_entry? Overwriting the old cache_entry is stylistically cleaner because if create_cache_entry steals ownership you're technically not allowed to access the old cache entry anymore right.

Reply:
Good point. Making the change.

Reply:
You haven't actually done this change yet